packages/main/src/components/protected/topbar/components/ProfilePicView.jsx

```diff
@@ -1,12 +1,15 @@
 import React, { useContext } from "react";
-import { TopbarContext } from "../context/Topbar";
-import { ProfileContext } from "../context/ProfileModal";
+// import { TopbarContext } from "../context/Topbar";
+import { TopbarContext } from "../context/topbar.context";
+// import { ProfileContext } from "../context/ProfileModal";
+import { ProfileContext } from "../context/profile-modal.context";
 
 import styles from "../styles/ProfilePicView.module.css";
 
 const ProfilePicView = () => {
   const { profilePicView, setProfilePicView } = useContext(TopbarContext);
   const { userProfileImage, user } = useContext(ProfileContext);
+  console.log(profilePicView);
```

Comment:
It is not good practice to push console.log to a codebase. Ensure you've removed it from your commits

Reply:
ive resolved this